tools/unlabel

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```

Comment:
BSD users will be quick to complain this should be:
```suggestion
#!/usr/bin/env bash
```

However, I also got caught out by this script expecting to be run from within `tools/` (other scripts here don't need that). Feel free to grab https://github.com/dra27/ocaml/commit/abe93c2ff7a61362440b198590c77654cf0244c9 which lifts that limitation.

Reply:
Thanks for both these, but I'm afraid a lengthy googling is not helping me learn how to cherry-pick across forks. The best i could find was this, which failed:

```
gorge:ocaml john$ git fetch git://github.com/dra27/ocaml
From git://github.com/dra27/ocaml
 * branch                HEAD       -> FETCH_HEAD
gorge:ocaml john$ git cherry-pick dd0cefa0287739e435dfc1ce691f867acd000945
fatal: bad object dd0cefa0287739e435dfc1ce691f867acd000945
```

Can you advise?

Reply:
The github interface seems to suggest the commit you want is part of the `john-trunk` branch, so `git fetch git://github.com/dra27/ocaml john-trunk` should fetch the commit you want and the cherry-pick command should then work as expected.

Reply:
(`git fetch git://github.com/dra27/ocaml dd0cefa0287739e435dfc1ce691f867acd000945` directly should work as well)

Reply:
Thanks for the hand-holding @lthls. All done.

Reply:
Sorry, that wasn't supposed to cause a Git nightmare! I tend to pull in additional complete remotes when doing this - so `git remote add johnwhitington https://github.com/johnwhitington/ocaml.git -f`. You then get everything in that user's fork (and can do `git fetch johnwhitington` to update the clone of it)